tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/ConfigurationStore.cs

```diff
@@ -33,7 +33,7 @@ public async Task<IRepositoryConfiguration> GetRepositoryConfigurationAsync(long
             try
             {
                 var client = await clientFactory.GetInstallationClientAsync(installationId, cancellationToken);
-                var searchResults = await client.Repository.Content.GetAllContents(repositoryId, "CHECKENFORCER");
+                var searchResults = await client.Repository.Content.GetAllContents(repositoryId, "eng/CHECKENFORCER");
```

Comment:
By the way I'd be fine with this being in the `.github` folder as well, my main goal is to try and keep this type of clutter out of the root of the repo.